### DISCUSSION TITLE
Wikijs connect via LDAPS to FreeIPA

### DISCUSSION THREAD

**Discussion by forgeOneHQ** [requarks/wiki#3503] [Error / Bug Report]
**Describe the bug**
Configuration of the LDAP / Active Directory Authentication over LDAPS.

Ports are configured correctly, I also tested LDAP/LDAPS to be sure that it is not just the certificate error.

I added the certificate via .bashrc for the user to run as systemd, also tried it manually and see what happened.
export NODE_EXTRA_CA_CERTS="/etc/pki/tls/certs/dc01.domain.tld.crt"

When I try to login with an FreeIPA User:
Frontend Error: "GraphQL error: Cannot return null for non-nullable field ResponseStatus.slug."

I enabled logging for debug, also used "silly" but no informationen in journalctl -u wiki.

On my Rocket.Chat app I don't have that issue.

System preferences:
Node: 14.14.0
NPM: 6.14.8
PostgreSQL: 12
OS: CentOS8
Wikijs: 2.5.159

**Comment by forgeOneHQ** [comment]
@NGPixel is there any possibility to debug it in a better way?

Just to help to deliver some more outputs for it.

**Comment by forgeOneHQ** [comment]
@NGPixel Issue also with the latest 2.5.170 version.

**Comment by Saphar** [comment]
This is opened for a long time, is there any news on this? I am facing the same issue.

I have a custom certificate to connect. Earlier I had the error TLS confidentiality required, but now it seems that is fixed and I get this same error.

**Reply by forgeOneHQ** [reply]
Do you use starttls instead of LDAPS? I try to make another run with ldap over tls and see what comes up. I can give more details later.
